integr8ly/tutorial-web-app
# PF4 Branch: Rename icons and file names

## Motivation
Cleanup icon references for Font Awesome 5. Fix previous concern regarding the use of `_all.scss` files for importing groups.

## Verification Steps

Build should run without errors. No errors for icons should appear in the console.

## Checklist:


## Progress
